urx/urrobot.py

```diff
@@ -87,6 +87,15 @@ def send_program(self, prog):
         self.logger.info("Sending program: " + prog)
         self.secmon.send_program(prog)
 
+    def send_custom_program(self, prog):
+        """
+        send a complete program using urscript to the robot
+        the program is executed immediatly and any runnning
+        program is interrupted
+        """
+        self.logger.info("Sending program: " + prog)
+        self.rtmon.send_program(prog)
```

Comment:
what is the difference between that method and the exisitng one? it looks like it is using another interface. Why?
give it an explicit name and write in doc string what is the difference from the other one